app/models/piece.rb

```
@@ -1,6 +1,12 @@
 class Piece < ActiveRecord::Base
   belongs_to :game
 
+
+  def image
+    ' '
```

Raise NotImplementedError instead of returning blank. A piece with no image isn't implemented